proto/buf/alpha/registry/v1alpha1/search.proto

```diff
@@ -86,6 +96,10 @@ message SearchRequest {
   uint32 page_size = 2;
   // The first page is returned if this is 0.
   uint32 page_token = 3;
+  // Empty list means show all. Supplying one or more enums will 
+  // limit the search to the only the requested resources. 
+  // Supplying all possible enums is equivalent to empty list of filters.
+  repeated SearchFilter filters = 4;
```

Comment:
With this in place, is the idea to remove `SearchUsers` etc?

Reply:
yes that's what i would want to do!